docs/guides/layers.md

```diff
@@ -84,7 +84,18 @@ or 3 less than the total number of dimensions of the layer, allowing you to
 browse volumetric timeseries data and other high dimensional data. See for
 example these cells undergoing mitosis in this volumetric timeseries:
 
-![napari viewer with an image of a cell undergoing mitosis. The scroll bar below the canvas controls the timeseries, allowing different stages of mitosis to be visible.](./images/mitosis.webm)
+```{raw} html
+<figure>
+  <video width="100%" controls autoplay loop muted playsinline>
+    <source src="../../_static/images/mitosis.webm" type="video/webm" />
+    <source src="../../_static/images/mitosis.mp4" type="video/mp4" />
+    <img src="../../_static/images/mitosis.png"
```

Comment:
```suggestion
    <source src="../_static/images/mitosis.webm" type="video/webm" />
    <source src="../_static/images/mitosis.mp4" type="video/mp4" />
    <img src="../_static/images/mitosis.png"
```
I think the path is just wrong. We're in `docs/guides` here so `_static` is just up one and over.